lib/exosuit/instance.rb

```diff
@@ -44,5 +44,14 @@ def self.all
     def self.running
       all.select { |i| i.state.name == 'running' }
     end
+
+    def self.latest_ubuntu_ami
+      ubuntu_image ||= Exosuit.ec2.images(
```

Comment:
I imagine your goal with `ubuntu_image ||=` is to cache the value of the variable. I don't believe a local variable will achieve that effect, I believe you need to use an instance variable. I'd also vote that we use the same name as the method name, e.g. `@latest_ubuntu_ami ||=`, which is kind of a convention I use everywhere.